### PR TITLE
fixed an issue with playbook save

### PR DESCRIPTION
**Pull request by YuvHayun**
## Related Issues
fixes: [CIAC-11111](https://jira-dc.paloaltonetworks.com/browse/CIAC-11111)

## Description
Fixed an issue where playbook saved due to a failed attempt to dump task object.

### CODE REVIEW

File: .changelog/4416.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue where playbook saved due to a failed attempt to dump task object.
```
Comment (JudahSchwartz) [inline .changelog/4416.yml:2]: do you mean failed to save?

File: demisto_sdk/commands/content_graph/objects/base_playbook.py [hunk 2]
```diff
@@ -206,7 +206,7 @@ def _client_upload_method(cls, client: demisto_client) -> Callable:
         return client.import_playbook
 
     def save(self):
-        super().save()
+        super().save(fields_to_exclude=["tasks"])
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/content_graph/objects/base_playbook.py:209]: why dont we want this to be saved?

File: demisto_sdk/commands/content_graph/objects/integration.py [hunk 3]
```diff
@@ -199,7 +199,7 @@ def match(_dict: dict, path: Path) -> bool:
         return False
 
     def save(self):
-        super().save()
+        super().save(fields_to_exclude=["params"])
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/content_graph/objects/integration.py:202]: why dont we want this to be saved?

**Review by JudahSchwartz — APPROVED** [review]

File: .changelog/4416.yml (hunk 1 above)
Comment (YuvHayun) [reply .changelog/4416.yml:2]: save means dumping the object.

File: demisto_sdk/commands/content_graph/objects/base_playbook.py (hunk 2 above)
Comment (YuvHayun) [reply demisto_sdk/commands/content_graph/objects/base_playbook.py:209]: we do, we just save it by parsing the object itself as part of the base_playbook save and not the super.

File: demisto_sdk/commands/content_graph/objects/integration.py (hunk 3 above)
Comment (YuvHayun) [reply demisto_sdk/commands/content_graph/objects/integration.py:202]: we do, we just save it by parsing the object itself as part of the integration save and not the super.

**Pull request merged**